packages/react-charting/src/utilities/colors.ts

```diff
@@ -1,30 +1,260 @@
-
-const QUALITATIVE_COLORS = Object.values(QualitativePalette);
-
-export const getNextColor = (index: number, offset: number = 0): string => {
-  return QUALITATIVE_COLORS[(index + offset) % QUALITATIVE_COLORS.length];
+type ColorVariants = {
+  shade50: string;
+  shade40: string;
+  shade30: string;
+  shade20: string;
+  shade10: string;
+  primary: string;
+  tint10: string;
+  tint20: string;
+  tint30: string;
+  tint40: string;
+  tint50: string;
+  tint60: string;
+};
+
+const cornflower: ColorVariants = {
+  shade50: '#0d1126',
+  shade40: '#182047',
+  shade30: '#2c3c85',
+  shade20: '#3c51b4',
+  shade10: '#4760d5',
+  primary: '#4f6bed',
+  tint10: '#637cef',
+  tint20: '#778df1',
+  tint30: '#93a4f4',
+  tint40: '#c8d1fa',
+  tint50: '#e1e6fc',
+  tint60: '#f7f9fe',
+};
+
+const hotPink: ColorVariants = {
+  shade50: '#240016',
+  shade40: '#44002a',
+  shade30: '#7f004e',
+  shade20: '#ad006a',
+  shade10: '#cc007e',
+  primary: '#e3008c',
+  tint10: '#e61c99',
+  tint20: '#ea38a6',
+  tint30: '#ee5fb7',
+  tint40: '#f7adda',
+  tint50: '#fbd2eb',
+  tint60: '#fef4fa',
+};
+
+const teal: ColorVariants = {
+  shade50: '#001516',
+  shade40: '#012728',
+  shade30: '#02494c',
+  shade20: '#026467',
+  shade10: '#037679',
+  primary: '#038387',
+  tint10: '#159195',
+  tint20: '#2aa0a4',
+  tint30: '#4cb4b7',
+  tint40: '#9bd9db',
+  tint50: '#c7ebec',
+  tint60: '#f0fafa',
+};
+
+const orchid: ColorVariants = {
+  shade50: '#16101d',
+  shade40: '#281e37',
+  shade30: '#4c3867',
+  shade20: '#674c8c',
+  shade10: '#795aa6',
+  primary: '#8764b8',
+  tint10: '#9373c0',
+  tint20: '#a083c9',
+  tint30: '#b29ad4',
+  tint40: '#d7caea',
+  tint50: '#e9e2f4',
+  tint60: '#f9f8fc',
+};
+
+const lightGreen: ColorVariants = {
+  shade50: '#031a02',
+  shade40: '#063004',
+  shade30: '#0b5a08',
+  shade20: '#0e7a0b',
+  shade10: '#11910d',
+  primary: '#13a10e',
+  tint10: '#27ac22',
+  tint20: '#3db838',
+  tint30: '#5ec75a',
+  tint40: '#a7e3a5',
+  tint50: '#cef0cd',
+  tint60: '#f2fbf2',
+};
+
+const lightBlue: ColorVariants = {
+  shade50: '#091823',
+  shade40: '#112d42',
+  shade30: '#20547c',
+  shade20: '#2c72a8',
+  shade10: '#3487c7',
+  primary: '#3a96dd',
+  tint10: '#4fa1e1',
+  tint20: '#65ade5',
+  tint30: '#83bdeb',
+  tint40: '#bfddf5',
+  tint50: '#dcedfa',
+  tint60: '#f6fafe',
+};
+
+const pumpkin: ColorVariants = {
+  shade50: '#200d03',
+  shade40: '#3d1805',
+  shade30: '#712d09',
+  shade20: '#9a3d0c',
+  shade10: '#b6480e',
+  primary: '#ca5010',
+  tint10: '#d06228',
+  tint20: '#d77440',
+  tint30: '#df8e64',
+  tint40: '#efc4ad',
+  tint50: '#f7dfd2',
+  tint60: '#fdf7f4',
+};
+
+const lime: ColorVariants = {
+  shade50: '#121b06',
+  shade40: '#23330b',
+  shade30: '#405f14',
+  shade20: '#57811b',
+  shade10: '#689920',
+  primary: '#73aa24',
+  tint10: '#81b437',
+  tint20: '#90be4c',
+  tint30: '#a4cc6c',
+  tint40: '#cfe5af',
+  tint50: '#e5f1d3',
+  tint60: '#f8fcf4',
+};
+
+const lilac: ColorVariants = {
```

Comment:
we don't need to duplicate defining these colors. It is already getting defined in the fluent token package. We need to understand how to take a dependency on it.
Meanwhile you can specify the # codes and mention the color code in comments.

Reply:
done